app/views/deploy_mailer/deployment_email.html.erb

```diff
@@ -1,3 +1,77 @@
+<style>
+.jira-issues, .jira-commits {
```

Comment:
Unfortunately for emails, you cannot use a `<style>` tag to add css styling to the HTML tags. In order to properly style a tag in an email, it must use the inline `style=""` attribute on the tags. So all of this needs to be moved to use inline styles rather than css styles.